### PR TITLE
Issuer Role

### PR DESCRIPTION
**Pull request by Sarahb-da**
Adds the issuer role, who can create asset tokens

### CODE REVIEW

**Review by dimitri-da — APPROVED** [review]
lgtm! you will need to rebase this on top of the model refactoring though which means that you will have to update your imports

File: ui/src/components/Issuer/Issuer.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import IssueAsset from './IssueAsset'
+import IssuedToken from './IssuedToken'
+
+type Props = {
+    onLogout: () => void;
+}
+
+const Issuer: React.FC<Props> = ({ onLogout }) => (
+    <Switch>
+        <Route path='/role/issuer/issue-asset'>
```
Comment (alexmatson-da) [inline ui/src/components/Issuer/Issuer.tsx:17]: I also used absolute paths at first, but found what I think is a good pattern for making them relative. Here in the issuer component, you can do 

```
const { path, url } = useRouteMatch();
```

Then, build paths off the path value like ``<Route path={`${path}/issue-asset`}>``

For the sidenav links, pass the `url` into the sidenav and then you can do the same thing - ``<Menu.Item to={`${url}/issue-asset`} ... >`` for example

And you can use just `${url}` with nothing added, which will appropriately resolve to `/role/issuer`

**Review by alexmatson-da — COMMENTED** [review]
Great work! One suggestion though

File: ui/src/components/Issuer/Issuer.tsx (hunk 1 above)
Comment (Sarahb-da) [reply ui/src/components/Issuer/Issuer.tsx:17]: ur right I will make that change - much better

**Review by alexmatson-da — APPROVED** [review]

**Pull request closed without merging**